content/browser/secure_embed_connector_impl.cc

```diff
@@ -159,6 +186,12 @@ void SecureEmbedConnectorImpl::SetFocusedFrameTree(
 
   // Update focused frame tree stored in the embedder.
   embedder_web_contents()->SetFocusedFrameTree(frame_tree_to_focus);
+  // The `frame_tree_to_focus` must belong to this WebContents
+  // or an inner WebContents in the subtree.
+  CHECK(ContainsOrIsFocusedWebContents(guest_web_contents_));
+
+  // Set the root frame as the focused frame.
+  frame_tree_to_focus->SetFocusedFrame(frame_tree_to_focus->root(), nullptr);
```

Comment:
The code seems to change the focused frame when setting focused frame tree, which doesn't seem to be the right thing to do. The similar code in WebContentsImpl::SetFocusedFrameTree only does this when frame_tree_to_focus is inner WebContents with some comments about unset focus for placeholder rfh.

We might want to test these scenarios: focus in embedder, click in an input element inside an OOPIF in embedder and start typing; similarly, create an OOPIF in embedder, with focus in embed, click into that OOFIF and start typing.